### PR TITLE
replacing zimbra: with : on all pages containing 'title'

### PR DESCRIPTION
**Pull request by akumarsynacor**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -4,7 +4,9 @@
 	<classpathentry kind="lib" path="build/dist/lib/coverage.jar"/>
 	<classpathentry kind="lib" path="build/dist/lib/resources.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="con" path="org.apache.ivyde.eclipse.cpcontainer.IVYDE_CONTAINER/?project=zm-selenium&amp;ivyXmlPath=ivy.xml&amp;confs=*"/>
 	<classpathentry kind="lib" path="build/dist/lib/ZimbraSelenium.jar"/>
+	<classpathentry kind="lib" path="/Users/kumara/work/eclipse-workspace/zm-mailbox/native/build/zm-native-8.8.0.1522920233.jar"/>
```
Comment (jiteshsojitra) [inline .classpath]: This probably by mistake came and not required to commit.

File: src/java/com/zimbra/qa/selenium/projects/ajax/tests/briefcase/document/DisplayDocument.java [hunk 2]
```diff
@@ -91,7 +91,7 @@ public void DisplayDocument_02() throws HarnessException {
 
 		// Create document item
 		DocumentItem document = new DocumentItem();
-		String docName = "test���test";
+		String docName = "test���test";
```
Comment (jiteshsojitra) [inline src/java/com/zimbra/qa/selenium/projects/ajax/tests/briefcase/document/DisplayDocument.java]: UTF format changed?

**Review by singhjitesh — APPROVED** [review]

File: .classpath (hunk 1 above)
Comment (akumarsynacor) [reply .classpath]: Changed back to original

File: src/java/com/zimbra/qa/selenium/projects/ajax/tests/briefcase/document/DisplayDocument.java (hunk 2 above)
Comment (akumarsynacor) [reply src/java/com/zimbra/qa/selenium/projects/ajax/tests/briefcase/document/DisplayDocument.java]: changed back to original

**Review by jiteshsojitra — APPROVED** [review]

**Pull request merged**